bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -0,0 +1,2135 @@
+                break;
+            case THING_TYPE_ID_RCD1570:
+                rotelModel = RotelModel.RCD1570;
+                break;
+            case THING_TYPE_ID_RCD1572:
+                rotelModel = RotelModel.RCD1572;
+                break;
+            case THING_TYPE_ID_RCX1500:
+                rotelModel = RotelModel.RCX1500;
+                break;
+            case THING_TYPE_ID_RDD1580:
+                rotelModel = RotelModel.RDD1580;
+                break;
+            case THING_TYPE_ID_RDG1520:
+            case THING_TYPE_ID_RT09:
+                rotelModel = RotelModel.RDG1520;
+                break;
+            case THING_TYPE_ID_RSP1576:
+                rotelModel = RotelModel.RSP1576;
+                break;
+            case THING_TYPE_ID_RSP1582:
+                rotelModel = RotelModel.RSP1582;
+                break;
+            case THING_TYPE_ID_RT11:
+                rotelModel = RotelModel.RT11;
+                break;
+            case THING_TYPE_ID_RT1570:
+                rotelModel = RotelModel.RT1570;
+                break;
+            case THING_TYPE_ID_T11:
+                rotelModel = RotelModel.T11;
+                break;
+            case THING_TYPE_ID_T14:
+                rotelModel = RotelModel.T14;
+                break;
+            default:
+                rotelModel = DEFAULT_MODEL;
+                break;
+        }
+
+        RotelThingConfiguration config = getConfigAs(RotelThingConfiguration.class);
+
+        RotelProtocol rotelProtocol = RotelProtocol.HEX;
+        if (config.protocol != null && !config.protocol.isEmpty()) {
+            try {
+                rotelProtocol = RotelProtocol.getFromName(config.protocol);
+            } catch (RotelException e) {
+            }
+        } else {
+            Map<String, String> properties = editProperties();
+            String property = properties.get(RotelBindingConstants.PROPERTY_PROTOCOL);
+            if (property != null && !property.isEmpty()) {
+                try {
+                    rotelProtocol = RotelProtocol.getFromName(property);
+                } catch (RotelException e) {
+                }
+            }
+        }
+        logger.debug("rotelProtocol {}", rotelProtocol.getName());
+
+        Map<RotelSource, String> sourcesLabels = new HashMap<>();
+
+        connector = new RotelSimuConnector(rotelModel, rotelProtocol, sourcesLabels);
+
+        if (rotelModel.hasVolumeControl()) {
+            maxVolume = rotelModel.getVolumeMax();
+            if (!rotelModel.hasDirectVolumeControl()) {
+                logger.info(
+                        "Set minValue to {} and maxValue to {} for your sitemap widget attached to your volume item.",
+                        minVolume, maxVolume);
+            }
+        }
+        if (rotelModel.hasToneControl()) {
+            maxToneLevel = rotelModel.getToneLevelMax();
+            minToneLevel = -maxToneLevel;
+            logger.info(
```

Comment:
Ideally, bass and treble should be percentage values as well, then such unusual logs would not be required.
But I leave it to you if you have a strong opinion about it.

Reply:
I am not convinced that a bass/treble adjustment should be shown as a percent. This is generally an adjustment around 0, 0 being the neutral state.

Reply:
Do you know if a Dimmer channel with a min value to -10 and a max value to +10 set in the state description would be correctly  handled by UIs ?
I can make a try. If it works, it could be a clean solution for models having direct set command for bass/treble (all models with tone control using the ASCII protocol).

Reply:
No, Dimmer is by definition a percentage, i.e. 0%-100%. So yes, ok for me to NOT use dimmer for that case. I'd just think that a clear min/max would be nicer than to spit out log messages for the user...